components/brave_rewards/browser/publisher_info_database_unittest.cc

```diff
@@ -115,7 +115,15 @@ class PublisherInfoDatabaseTest : public ::testing::Test {
           base::KEEP_WHITESPACE,
           base::SPLIT_WANT_NONEMPTY);
 
-      data = base::JoinString(split, "\n") + "\n";
+      if (split.size() > 1) {
```

Comment:
Have we tried normalizing the data files, i.e. saving on macOS so that all files have new lines instead of CRLF?